test/test_util.ts

```diff
@@ -61,10 +61,25 @@ export function testWithExpectedDate(chrono: ChronoLike, text: string, expectedD
     });
 }
 
-export function testUnexpectedResult(chrono: ChronoLike, text: string, refDate?: Date) {
+export function testUnexpectedResult(chrono: ChronoLike, text: string, refDate?: Date);
+export function testUnexpectedResult(
+    chrono: ChronoLike,
+    text: string,
+    refDate?: Date,
+    optionOrCheckResult?: ParsingOption | CheckResult
+);
+export function testUnexpectedResult(
+    chrono: ChronoLike,
+    text: string,
+    refDate?: Date,
+    optionOrCheckResult?: ParsingOption | CheckResult
```

Comment:
Nit: CheckResult is not really supported here, right? so `options? : ParsingOption`?